server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CachedBlobStore.java

```diff
@@ -60,8 +60,7 @@ public InputStream read(BucketName bucketName, BlobId blobId) throws ObjectStore
 
         return Mono.just(bucketName)
             .filter(backend.getDefaultBucketName()::equals)
-            .flatMap(ignored ->
-                Mono.from(cache.read(blobId))
+            .flatMap(ignored -> Mono.from(cache.read(blobId))
```

Comment:
That's not a rebase but a fixup! no?

Reply:
I think it acceptable for rebase, wdyt?

Reply:
We can squash it upon merge? (but I prefer a fixup! as it makes quick pre merge history rework easier)

Reply:
Yeah, i will change it to fixup